### PR TITLE
Added MakeHuman Exception License

### LINKED ISSUE #816: Add Makehuman license exception?
See http://www.makehuman.org/license.php and  http://www.makehuman.org/license_explanation.php

> As a special and limited exception, the copyright holders of the MakeHuman assets grants the option 
to use CC0 1.0 Universal as published by the Creative Commons, either version 1.0 of the License, or (at your option) any later version, as a license for the MakeHuman characters exported under the conditions that a) The assets were bundled in an export that was made using the file export functionality inside an OFFICIAL and UNMODIFIED version of MakeHuman and/or b) the asset solely consists of a 2D binary image in PNG, BMP or JPG format.

### PR DESCRIPTION
**Pull request by SaravananOffl**
Fixes #816 
Signed-off-by: Saravanan G <sgoffl1603@gmail.com>

### CODE REVIEW

**Comment by SaravananOffl** [comment]
@pombredanne I wrote the LICENSE file for the exception and a RULE file for tests but the tests are not passing .Could you help me to rectify the mistake?

File: src/licensedcode/data/licenses/make-human-exception.LICENSE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+MakeHuman output GPL exception
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/make-human-exception.LICENSE]: Can you format this to 80 characters lines?

File: src/licensedcode/data/licenses/make-human-exception.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+key: make-human-exception
+short_name: MakeHuman License
+name: MakeHuman License
+category: Permissive
+owner: MakeHuman
+homepage_url: http://www.makehuman.org/index.php
+text_urls:
+    - http://www.makehuman.org/license.php
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/make-human-exception.yml]: clean and neat! :+1:

File: src/licensedcode/data/rules/make-human-exception.RULE [hunk 3]
```diff
@@ -0,0 +1,11 @@
+MakeHuman output GPL exception
```
Comment (pombredanne) [inline src/licensedcode/data/rules/make-human-exception.RULE]: Do not add a rule that is the same as the license. This is an error condition. Every license is automatically made in a rule.... so rules are for variants

Comment (SaravananOffl) [reply src/licensedcode/data/rules/make-human-exception.RULE]: So ,This file is not needed .Right?

Comment (pombredanne) [reply src/licensedcode/data/rules/make-human-exception.RULE]: correct

Comment (pombredanne) [reply src/licensedcode/data/rules/make-human-exception.RULE]: and its yml buddy too.

**Comment by SaravananOffl** [comment]
@pombredanne I've modified according to your instructions!

**Comment by pombredanne** [comment]
With a test this is will be perfect.

**Comment by SaravananOffl** [comment]
@pombredanne I've added test files also! Local tests ran perfectly!

**Comment by pombredanne** [comment]
Perfect. Thank you very much! merging.

**Pull request merged**